openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -897,6 +897,13 @@ def hide_progress_tab(self):
         """
         return self._original_course.hide_progress_tab
 
+    @property
+    def disable_progress_graph(self):
+        """
+        TODO: move this to the model.
+        """
+        return self._original_course.disable_progress_graph
+
```

Comment:
This is not required, you can directly access this attribute via `block.disable_progress_graph` [here](https://github.com/RafayGhafoor/edx-platform/blob/57ede0241092231dce247a6cf98f9d456aadc39a/lms/djangoapps/course_home_api/progress/views.py#L232-L232) once you move to progress API view.